packages/editor/package.json

```diff
@@ -37,7 +37,7 @@
     "watch:client": "webpack-dev-server --mode development --hot --inline",
     "watch:server": "run-p watch:server:*",
     "watch:server:start": "nodemon -r dotenv/config --inspect=0.0.0.0:9230 ./dist/server/index.js",
-    "watch:server:build": "tsc -b ./src/shared ./src/server --watch",
+    "watch:server:build": "tsc -b ./src/shared ./src/server ./src/client --watch",
```

Comment:
why?

Reply:
For me the hot reload of the editor did not work without this.... Does anyone has the same problem?